### ISSUE TITLE
Highlighting text in /connect doesn't work

### ISSUE THREAD

**Issue by Zhao-Andy** [thepracticaldev/dev.to#568]
### Current Behavior
Highlighting text in /connect chat channels does not work as intended. Here's a `.gif` example:
![screen recording 2018-08-31 at 02 13 pm](https://user-images.githubusercontent.com/17884966/44929207-27090600-ad28-11e8-913f-9b4bf4de3f7e.gif)

It's a bit inconsistent, but for the most part, it deselects the highlighted text shortly after being selected.
### Expected Behavior
Highlighting text selects the text and keeps it selected.

### Additional Info (device/browser, helpful links)
Doesn't work on macOS Chrome + Firefox.

**Comment by Antrikshy** [comment]
I'm trying to reproduce this in my dev environment. What do I need to do to get a mock chat going? Sending messages gets me this printing out in the backend:

    ActiveRecord::RecordNotFound - Couldn't find ChatChannel with 'id'=:
      app/models/message.rb:91:in `channel_permission'
      app/controllers/messages_controller.rb:10:in `create'

**Comment by Zhao-Andy** [comment]
Hey @Antrikshy, you can run this in `rails console`:
```ruby
# assuming you're the most recent user created
ChatChannel.create_with_users([User.last, User.first])
```

**Comment by Antrikshy** [comment]
All right, I see a chat now. Sending messages now gets me:

```
Processing by MessagesController#create as JSON [controller_call]
Pusher::HTTPError (getaddrinfo: nodename nor servname provided, or not known (api-Optional.pusher.com:443) (SocketError)) [error]

SocketError - getaddrinfo: nodename nor servname provided, or not known (api-Optional.pusher.com:443):
  app/controllers/messages_controller.rb:12:in `create'
```

**Comment by Zhao-Andy** [comment]
Gotcha, you'll need a Pusher app to use chat. You can create a new app for free by creating an account here: https://dashboard.pusher.com/accounts/sign_up

There are two keys, `PUSHER_KEY` and `PUSHER_SECRET`. It should be pretty straightforward, but feel free to ask questions if you run into any issues.

**Comment by maestromac** [comment]
Resolved with #1311
